### PR TITLE
Change to updated endpoint

### LINKED ISSUE #33633: [Bug]: `aws_chime_voice_connector` errors with `ForbiddenException` in new AWS accounts
### Terraform Core Version

1.5.7

### AWS Provider Version

5.17.0

### Affected Resource(s)

* aws_chime_voice_connector

### Expected Behavior

Voice Connector should have been provisioned successfully

### Actual Behavior

AWS API throws 403 ForbiddenException, despite the IAM user having full `AdministratorAccess` policy directly attached.

### Relevant Error/Panic Output Snippet

```shell
2023-09-26T16:22:26.9596528Z aws_chime_voice_connector.vc: Creating...
2023-09-26T16:22:26.9598062Z ╷
2023-09-26T16:22:26.9598984Z │ Error: creating Chime Voice connector: ForbiddenException: Not authorized to call Chime SDK with Account Id xxxxxxxx
2023-09-26T16:22:26.9599285Z │ {
2023-09-26T16:22:26.9599612Z │   RespMetadata: {
2023-09-26T16:22:26.9599782Z │     StatusCode: 403,
2023-09-26T16:22:26.9600047Z │     RequestID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
2023-09-26T16:22:26.9600216Z │   },
2023-09-26T16:22:26.9600372Z │   Code_: "Forbidden",
2023-09-26T16:22:26.9600660Z │   Message_: "Not authorized to call Chime SDK with Account Id xxxxxxxxxxxx"
2023-09-26T16:22:26.9600835Z │ }
2023-09-26T16:22:26.9600973Z │
2023-09-26T16:22:26.9601160Z │   with aws_chime_voice_connector.vc,
2023-09-26T16:22:26.9601416Z │   on main.tf line 5, in resource "aws_chime_voice_connector" "vc":
2023-09-26T16:22:26.9601683Z │    5: resource "aws_chime_voice_connector" "vc" {
2023-09-26T16:22:26.9601837Z │
2023-09-26T16:22:26.9602143Z ╵
2023-09-26T16:22:27.4415035Z Operation failed: failed running terraform apply (exit 1)
```

### Terraform Configuration Files

`main.tf`:
```hcl
resource "aws_chime_voice_connector" "vc" {
  name               = var.voice_connector_name
  aws_region         = var.voice_connector_aws_region
  require_encryption = true
  tags               = var.voice_connector_tags
}
```

`variables.tf`
```hcl
variable "voice_connector_name" {
  type        = string
  description = "Name of Voice Connector resource"
}

variable "voice_connector_aws_region" {
  type        = string
  description = "AWS region name"
}

variable "voice_connector_tags" {
  type        = map(any)
  description = "AWS tags to be applied to voice connectors"
}
```

### Steps to Reproduce

1. Create AWS Account that hasn't previously used AWS Chime SDK
2. Create IAM user and directly attach "AdministratorAccess" policy
3. `terraform apply` to attempt creating `aws_chime_voice_connector`
4. Terraform errors during apply with `Error: creating Chime Voice connector: ForbiddenException: Not authorized to call Chime SDK with Account Id xxxxxxxxxxxx`

### Debug Output

_No response_

### Panic Output

_No response_

### Important Factoids

The AWS Chime team working with us to set up this new service suspects that this issue is because the AWS Terraform Provider is still using the legacy Chime API that has been deprecated and no longer accepts requests from new accounts. I would love to put someone from the provider dev team in touch with them to help resolve.

### References

_No response_

### Would you like to implement a fix?

No

### PR DESCRIPTION
**Pull request by nam054**
### Description

Updates the deprecated chime endpoints

### Relations

Relates #2697
Closes #33633

### Output from Acceptance Testing

```console
> make testacc TESTS=TestAccChimeVoiceConnector_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnector_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnector_basic
--- PASS: TestAccChimeVoiceConnector_basic (19.40s)
=== RUN   TestAccChimeVoiceConnector_disappears
--- PASS: TestAccChimeVoiceConnector_disappears (13.30s)
=== RUN   TestAccChimeVoiceConnector_update
--- PASS: TestAccChimeVoiceConnector_update (30.13s)
=== RUN   TestAccChimeVoiceConnector_tags
--- PASS: TestAccChimeVoiceConnector_tags (42.86s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      108.768s

> make testacc TESTS=TestAccChimeVoiceConnectorGroup_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnectorGroup_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnectorGroup_basic
--- PASS: TestAccChimeVoiceConnectorGroup_basic (19.55s)
=== RUN   TestAccChimeVoiceConnectorGroup_disappears
--- PASS: TestAccChimeVoiceConnectorGroup_disappears (16.78s)
=== RUN   TestAccChimeVoiceConnectorGroup_update
--- PASS: TestAccChimeVoiceConnectorGroup_update (31.20s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      70.796s

> make testacc TESTS=TestAccChimeVoiceConnectorLogging_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnectorLogging_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnectorLogging_basic
--- PASS: TestAccChimeVoiceConnectorLogging_basic (20.27s)
=== RUN   TestAccChimeVoiceConnectorLogging_disappears
--- PASS: TestAccChimeVoiceConnectorLogging_disappears (17.04s)
=== RUN   TestAccChimeVoiceConnectorLogging_update
--- PASS: TestAccChimeVoiceConnectorLogging_update (31.25s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      71.635s

> make testacc TESTS=TestAccChimeVoiceConnectorOrigination_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnectorOrigination_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnectorOrigination_basic
--- PASS: TestAccChimeVoiceConnectorOrigination_basic (19.45s)
=== RUN   TestAccChimeVoiceConnectorOrigination_disappears
--- PASS: TestAccChimeVoiceConnectorOrigination_disappears (16.30s)
=== RUN   TestAccChimeVoiceConnectorOrigination_update
--- PASS: TestAccChimeVoiceConnectorOrigination_update (30.79s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      69.592s

> make testacc TESTS=TestAccChimeVoiceConnectorTermination_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnectorTermination_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnectorTermination_basic
--- PASS: TestAccChimeVoiceConnectorTermination_basic (19.63s)
=== RUN   TestAccChimeVoiceConnectorTermination_disappears
=== PAUSE TestAccChimeVoiceConnectorTermination_disappears
=== RUN   TestAccChimeVoiceConnectorTermination_update
=== PAUSE TestAccChimeVoiceConnectorTermination_update
=== CONT  TestAccChimeVoiceConnectorTermination_disappears
=== CONT  TestAccChimeVoiceConnectorTermination_update
--- PASS: TestAccChimeVoiceConnectorTermination_disappears (17.18s)
--- PASS: TestAccChimeVoiceConnectorTermination_update (32.78s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      55.491s

> make testacc TESTS=TestAccChimeVoiceConnectorTerminationCredentials_ PKG=chime
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChimeVoiceConnectorTerminationCredentials_'  -timeout 360m
=== RUN   TestAccChimeVoiceConnectorTerminationCredentials_basic
--- PASS: TestAccChimeVoiceConnectorTerminationCredentials_basic (20.35s)
=== RUN   TestAccChimeVoiceConnectorTerminationCredentials_disappears
--- PASS: TestAccChimeVoiceConnectorTerminationCredentials_disappears (16.70s)
=== RUN   TestAccChimeVoiceConnectorTerminationCredentials_update
--- PASS: TestAccChimeVoiceConnectorTerminationCredentials_update (29.43s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      69.601s
...
```

### CODE REVIEW

File: internal/service/chime/voice_connector_group_test.go [hunk 1]
```diff
@@ -20,17 +20,17 @@ import (
...
 func TestAccChimeVoiceConnectorGroup_basic(t *testing.T) {
 	ctx := acctest.Context(t)
-	var voiceConnectorGroup *chime.VoiceConnectorGroup
+	var voiceConnectorGroup *chimesdkvoice.VoiceConnectorGroup
 
 	vcgName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
 	resourceName := "aws_chime_voice_connector_group.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```
Comment (jar-b) [inline internal/service/chime/voice_connector_group_test.go:28]: Were these meant to be moved into a `_serial` function?

**Review by jar-b — COMMENTED** [review]
Overall looks good - just one clarification on the removal of parallel testing. If these cannot be safely run in parallel we can use a single `_serial` test at the package level and rename the existing tests such that they aren't exported and run directly, but executed sequentially in the parent test function instead.

File: internal/service/chime/voice_connector_group_test.go (hunk 1 above)
Comment (nam054) [reply internal/service/chime/voice_connector_group_test.go:40]: Yes - added them now

**Comment by johnsonaj** [comment]
Do all of these voice connector type resources require a aws_chime_voice_connector`? If yes, it might be worth putting them in one big serial test block since the limit on this service endpoint seems to be 3 by default. Having them as individual test will most likely cause errors when running in CI

**Comment by nam054** [comment]
Ah I see. Yes, they require the `aws_chime_voice_connector` 
Would that go in a separate file? And we'd need to export the tests again right?

**Review by jar-b — APPROVED** [review]
LGTM 🎉 

```console
% make testacc PKG=chime TESTS=TestAccChime_serial
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/chime/... -v -count 1 -parallel 20 -run='TestAccChime_serial'  -timeout 360m

--- PASS: TestAccChime_serial (1156.25s)
    --- PASS: TestAccChime_serial/VoiceConnector (173.15s)
        --- PASS: TestAccChime_serial/VoiceConnector/basic (29.86s)
        --- PASS: TestAccChime_serial/VoiceConnector/disappears (24.07s)
        --- PASS: TestAccChime_serial/VoiceConnector/update (49.47s)
        --- PASS: TestAccChime_serial/VoiceConnector/tags (69.74s)
    --- PASS: TestAccChime_serial/VoiceConnectorGroup (179.66s)
        --- PASS: TestAccChime_serial/VoiceConnectorGroup/basic (46.87s)
        --- PASS: TestAccChime_serial/VoiceConnectorGroup/disappears (46.62s)
        --- PASS: TestAccChime_serial/VoiceConnectorGroup/update (86.16s)
    --- PASS: TestAccChime_serial/VoiceConnectorLogging (191.66s)
        --- PASS: TestAccChime_serial/VoiceConnectorLogging/basic (57.77s)
        --- PASS: TestAccChime_serial/VoiceConnectorLogging/disappears (51.33s)
        --- PASS: TestAccChime_serial/VoiceConnectorLogging/update (82.56s)
    --- PASS: TestAccChime_serial/VoiceConnectorOrigination (186.40s)
        --- PASS: TestAccChime_serial/VoiceConnectorOrigination/basic (46.86s)
        --- PASS: TestAccChime_serial/VoiceConnectorOrigination/disappears (48.02s)
        --- PASS: TestAccChime_serial/VoiceConnectorOrigination/update (91.52s)
    --- PASS: TestAccChime_serial/VoiceConnectorTermination (178.57s)
        --- PASS: TestAccChime_serial/VoiceConnectorTermination/basic (51.01s)
        --- PASS: TestAccChime_serial/VoiceConnectorTermination/disappears (46.98s)
        --- PASS: TestAccChime_serial/VoiceConnectorTermination/update (80.58s)
    --- PASS: TestAccChime_serial/VoiceConnectorTerminationCredentials (246.81s)
        --- PASS: TestAccChime_serial/VoiceConnectorTerminationCredentials/basic (68.26s)
        --- PASS: TestAccChime_serial/VoiceConnectorTerminationCredentials/disappears (65.56s)
        --- PASS: TestAccChime_serial/VoiceConnectorTerminationCredentials/update (112.99s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/chime      1159.337s
```

**Comment by jeffhuenemann** [comment]
@nam054 Thank you for the work on this! Just checking; I see this added by the bot to the v5.22.0 milestone, but don't see it in changelog - is that the version where this will go in? Any idea on a release date?

**Pull request merged**